common.mk

```diff
@@ -0,0 +1,33 @@
+ARCH ?= x86
```

Comment:
How does this work with cargo and the Rust compilation?

Reply:
cargo has its own env variables e.g. `CARGO_CFG_TARGET_ARCH`. It should not be affected by this. 
I think your point is a bit more than that. I think I got it, but can you please elaborate a bit here so we're sure we're not talking about different things? :)

Reply:
My question is can `ARCH` be set to something different and would that make the end result (BPF+Rust) compatible with that non x86 arch?

Reply:
> this is for `PT_REGS_PARMx()` as they depend on the target.

I see, makes sense.

>  In general cross-compilation support should be possible, but should require pointing to the right backends (should not be a big deal).

Yeah that's fine, we don't support it for now. So maybe instead of `ARCH ?= ` we should use `ARCH = ` to make it clear we don't support anything else. Also a comment might be worth it (that some macros depends on this being set).